### PR TITLE
Changed more double-quotes to single

### PR DESCRIPTION
**Pull request by Quix0r**
More changes from double-quote to single.

### CODE REVIEW

**Comment by Quix0r** [comment]
````
 - bin/daemon.php
License headers are missing or wrong 
````
The header is there.

File: src/Database/Database.php [hunk 1]
```diff
@@ -825,6 +825,7 @@ public function e(string $sql): bool
 	 *
 	 * @return boolean Are there rows for that condition?
 	 * @throws \Exception
+	 * @todo Get rid of the DBStructure::existsTable() invocation
```
Comment (MrPetovan) [inline src/Database/Database.php]: Can you please expand on the why or provide a contact information should anyone require more information from you?

**Review by MrPetovan — CHANGES_REQUESTED** [review]
Looking good, almost there!

**Comment by MrPetovan** [comment]
> ```
>  - bin/daemon.php
> License headers are missing or wrong 
> ```
> 
> The header is there.

The script checks the first line for the PHP opening tag. Since we have the hashbang instead, this check fails, please disregard.

File: src/Database/Database.php (hunk 1 above)
Comment (Quix0r) [reply src/Database/Database.php]: This invocation is redundant and is just wrapped by `DBA::exists()`.

**Comment by nupplaphil** [comment]
> > ```
> >  - bin/daemon.php
> > License headers are missing or wrong 
> > ```
> > 
> > The header is there.
> 
> The script checks the first line for the PHP opening tag. Since we have the hashbang instead, this check fails, please disregard.

"Somebody" will Check it soon *gg*

File: src/Database/Database.php (hunk 1 above)
Comment (MrPetovan) [reply src/Database/Database.php]: Thank you, I meant in the code itself. 😄

Comment (Quix0r) [reply src/Database/Database.php]: For me, I differentiate between no table created and no records found in an existing table. `DBA::exists()` makes both cases the same. We should explicitly test if no table is there (incomplete installation/update) or if no record in an existing table was found.

Comment (MrPetovan) [reply src/Database/Database.php]: Thank you for the further elaboration, I just meant that you should ass the following to the `@todo` comment you added in this PR:

> This invocation is redundant and is just wrapped by `DBA::exists()`.

**Review by MrPetovan — APPROVED** [review]

**Comment by Quix0r** [comment]
If that invocation is now dumped, only `DBATest->testExists()`  needs to have 2nd test (on non-existing table) being removed.

**Pull request merged**